packages/libraries/cli/README.md

```diff
@@ -101,7 +101,7 @@ _See code:
 
 ## `hive artifact:fetch`
 
-fetch artifacts from the CDN
+fetch artifacts from the CDN. Note that the path shown in UI already contains `artifact`.
```

Comment:
This is actually set in code, and not in README: (this file is generated)

https://gitkraken.dev/link/dnNjb2RlOi8vZWFtb2Rpby5naXRsZW5zL2xpbmsvci9kYzNiYzFlYzg3ODJmMDdiYjViYzkxMjYyMDQ2Njk1NTA5NThjZGY5L2YvcGFja2FnZXMvbGlicmFyaWVzL2NsaS9zcmMvY29tbWFuZHMvYXJ0aWZhY3QvZmV0Y2gudHM%2FdXJsPWdpdCU0MGdpdGh1Yi5jb20lM0FrYW1pbGtpc2llbGElMkZncmFwaHFsLWhpdmUuZ2l0?origin=gitlens